runtime/service/src/main/java/org/apache/polaris/service/events/json/mixins/ObjectMapperFactory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.polaris.service.events.json.mixins;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.polaris.service.events.IcebergRestCatalogEvents;
+import org.apache.polaris.service.events.PolarisEvent;
+
+public class ObjectMapperFactory {
+  private ObjectMapperFactory() {}
+
+  public static ObjectMapper create() {
```

Comment:
You should not create an ObjectMapper like this, but rather use CDI.

1. If possible, let's use the main ObjectMapper provided by Quarkus. You can customize it by adding your customizations in the `org.apache.polaris.service.config.PolarisIcebergObjectMapperCustomizer` bean.
2. If it's not possible to reuse the main ObjectMapper (for example, because of incompatible configuration), then you should produce a new ObjectMapper bean.

I realize that the old version of `AwsCloudWatchEventListener` was also doing it wrong, so let's seize the opportunity to fix this.

Reply:
I agree with the CDI based approach. I found option #2 to be better feasible. 

Let's seize the opportunity!